src/main/java/org/cactoos/list/Immutable.java

```diff
@@ -55,10 +66,49 @@
 
     /**
      * Ctor.
-     * @param src Source
+     * @param items Source array
+     */
+    @SafeVarargs
+    public Immutable(final T... items) {
+        this(new IterableOf<>(items));
```

Comment:
@iakunin please, use `CollectionOf` instead of `IterableOf`.

Reply:
@fabriciofx fixed